src/azul/logging.py

```diff
@@ -6,9 +6,45 @@
 )
 
 
+class LambdaLoggerFilter(logging.Filter):
+
+    def __init__(self, app: AzulChaliceApp = None):
+        try:
+            # From https://github.com/aws/aws-lambda-base-images/tree/python3.8
+            # Imports global variable present in lambda environment
+            from bootstrap import (  # noqa:AZUL141
+                _GLOBAL_AWS_REQUEST_ID,
+            )
+        except ImportError:
+            # Block exists for local development/lack of aws_request_id variable
+            self.aws_request_id = "local"
+        else:
+            assert app.lambda_context.aws_request_id == _GLOBAL_AWS_REQUEST_ID
+            self.aws_request_id = _GLOBAL_AWS_REQUEST_ID
+
+    def filter(self, record):
+        record.aws_request_id = self.aws_request_id
+        return True
+
+
+log_format = '[%(levelname)s]\t%(asctime)s.%(msecs)03dZ\t%(name)s\t%(aws_request_id)s\t%(message)s\n'
+log_date_format = '%Y-%m-%dT%H:%M:%S'
+
+
 def configure_app_logging(app: AzulChaliceApp, *loggers):
     _configure_log_levels(app.log, *loggers)
 
+    root_logger = logging.getLogger()
+    root_formatter = logging.Formatter(log_format, log_date_format)
+    if len(root_logger.handlers) != 0:
```

Comment:
If multiple handlers can exist, how can we ensure that the one that we are modifying is the StreamHandler that we are looking for? Are there other inspections to the handler that we can assert?